ReactViewControl/ReactViewRender.cs

```diff
@@ -135,9 +135,9 @@ private enum LoadStatus {
         }
 
         private void InternalLoadComponent(IViewModule component, string frameName) {
-            var source = NormalizeUrl(component.JavascriptSource);
-            var baseUrl = ToFullUrl(VirtualPathUtility.GetDirectory(source));
-            var urlSuffix = cacheInvalidationTimestamp != null ? "t=" + cacheInvalidationTimestamp : null;
+            var source = ToFullUrl(NormalizeUrl(component.MainSource));
+            var originalSourceFolder = ToFullUrl(NormalizeUrl(component.OriginalSourceFolder));
+            var externalScripts = component.ExternalSources.Select(s => ToFullUrl(NormalizeUrl(s))).ToArray();
```

Comment:
cacheInvalidationTimestamp  went away? Did you remove the field?

Reply:
Yes. Isn't being used in loadComponent. The previous usage was:

`require.config({	
                paths: paths,	
                baseUrl: baseUrl,	
                urlArgs: cacheInvalidationSuffix	
            });`

Do we still need it?

Reply:
Nope, was used primarly for hot reload. But I didn't see the field removal, thats why I asked